tests/optim/param/test_transforms.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python3
+import unittest
+
+import torch
+import torch.nn.functional as F
+
+from captum.optim._param.image import transform
+from tests.helpers.basic import BaseTest
+

```

Comment:
If some of these transformations have numpy equivalents, would it be possible to assert our implementations against those equivalents ?

Reply:
It may be possible to implement some of the transforms in NumPy, but for the moment the current tests should suffice. I'll look into having NumPy transforms for testing in a future PR.

Reply:
Yeah, it definitely seems like it would be worthwhile to have NumPy or SciPy versions of our transforms for comparison in the future!

Reply:
Do you think we should add the NumPy / SciPy comparison tests in addition to the current tests, or replace them with the comparison tests?

Reply:
I've replaced the RandomSpatialJitter, CenterCrop, ToRGB, and BlendAlpha tests with NumPy comparion tests. The TestIgnoreAlpha and TestRandSelect tests don't require NumPy versions. 

I'm still looking at using NumPy for the TestRandomScale and TestGaussianSmoothing tests, but I'm leaving them for a future PR.

Reply:
I've recreated FFTImage in NumPy for testing now as well. The other Image classes and functions are not very straight forward to recreate in NumPy, so we can leave them for a future PR as well.